plugins/elanfp/fu-elanfp-device.c

```diff
@@ -403,7 +404,21 @@ fu_elanfp_device_write_firmware(FuDevice *device,
 		return FALSE;
 	if (!fu_elanfp_device_write_payload(self, payload, fu_progress_get_child(progress), error))
 		return FALSE;
+
+	/* hardware reset */
+	if (!fu_elanfp_device_do_xfer(self,
+				      (guint8 *)&usb_buf,
```

Comment:
ohh hang on -- you mean `usb_buf`, no? i.e. no `&`?

Reply:
I am follow line 230 to set command.

Reply:
okay, if it works I guess :) Can you rebase against master and force the branch please, I've fixed the CI issue. If you push to origin instead in the future I can fix this kind of thing for you myself :)

Reply:
Yes, I have test it. The function is works. Thanks for your help.